### PR TITLE
Whitepaper edits

### PR DESCRIPTION
**Pull request by rgbkrk**
As I read through, I made some edits to fix typos/grammar.

### CODE REVIEW

File: docs/whitepaper.md [hunk 1]
```diff
@@ -14,17 +14,17 @@ The broad adoption of computers in scientific research has made it possible to s
...
 
-To close the data divide, we must build tools that allow researchers to be able to focus more on scientific discovery and less on integrating disparate data. We introduce Dat, a version-controlled distributed database and data tool that has user interface of a version control system (VCS). Key features of dat include the ability to *track changes* in tabular and binary formats; *supply access points* across peer-to-peer internal and public networks; *create historical checkpoints* with metadata; *replicate* quickly at a given checkpoint; and *encourage forking* of data streams.
+To close the data divide, we must build tools that allow researchers to be able to focus more on scientific discovery and less on integrating disparate data. We introduce Dat, a version-controlled distributed database and data tool that has the user interface of a version control system (VCS). Key features of dat include the ability to *track changes* in tabular and binary formats; *supply access points* across peer-to-peer internal and public networks; *create historical checkpoints* with metadata; *replicate* quickly at a given checkpoint; and *encourage forking* of data streams.
```
Comment (rgbkrk) [inline docs/whitepaper.md]: > We introduce Dat, a version-controlled distributed database and data tool that has the user interface of a version control system (VCS).

This seems longer than it needs to be. What about "Distributed version control system for data"? I recognize this is more complete, but this is almost the same definition of git.

**Comment by rgbkrk** [comment]
This paper is wonderful and I'm extremely excited about the future of dat.

**Comment by karissa** [comment]
thanks @rgbkrk !

File: docs/whitepaper.md (hunk 1 above)
Comment (karissa) [inline docs/whitepaper.md]: Yeah, you're right

**Pull request merged**